com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1699,5 +1701,151 @@ internal void ApprovedPlayerSpawn(ulong clientId, uint playerPrefabHash)
                 NetworkMetrics.TrackObjectSpawnSent(clientPair.Key, ConnectedClients[clientId].PlayerObject, size);
             }
         }
+
```

Comment:
Poor NetworkManager is so large already....could this live in its own file?  Preferably in the Editor folder?

Reply:
I will see what I can do about removing the editor related code from NetworkManager into some other class within the NGO editor assembly.

Reply:
Ok, I see the update, but originally imagined like all this code from line 1705 to 1785 would be hoisted to `NetworkManagerHelper.cs`